system/lib/libcxxabi/src/cxa_exception.cpp

```diff
@@ -789,9 +789,9 @@ __cxa_rethrow_primary_exception(void* thrown_object)
         // In debug mode, call a JS library function to use
         // WebAssembly.Exception JS API, which enables us to include stack
         // traces
-        __throw_exception_with_stack_trace(&exception_header->unwindHeader);
+        __throw_exception_with_stack_trace(&dep_exception_header->unwindHeader);
 #else
-        _Unwind_RaiseException(&exception_header->unwindHeader);
+        _Unwind_RaiseException(&dep_exception_header->unwindHeader);
```

Comment:
It looks like this reverts this line to match upstream again?   If so, LGTM.  I wonder how this change changed in the first place

Reply:
> It looks like this reverts this line to match upstream again?

Yes

> I wonder how this change changed in the first place

In #20372, I copied the code from `__cxa_throw` and forgot to modify it 🥲